packages/components/pages/f-loyalty/src/tenants/en-NZ.js

```diff
@@ -33,12 +33,13 @@ const messages = {
         loginButtonText: 'Log in'
     },
     howItWorks: {
+        imagePath: 'https://just-eat-prod-eu-res.cloudinary.com/image/upload/v1630068495/Experiments/Homeweb-Coreweb/',
```

Comment:
Is this correct? Should we have a NZ-local domain, or does cloudinary handle this by having different DNS responses for requests in different geolocales?